core/packages/contracts/test/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
ja, I was getting weird errors in test files (eg. when loading JSON test data) about not having settings that were in the base tsconfig. Figured that came from the test files not being included in the base tsconfig, but I've tweaked it to include them.

Please let me know if this has worked for you, otherwise let's handle it later.

Reply:
We can't include the test files in the compilation though. Only the [auto-generated typescript bindings](https://github.com/dethcrypto/TypeChain) for our contracts should be included (the files under `src`), as they will be packaged and uploaded to NPM eventually.

This is why I suggest having two different tsconfigs, one for editors, and one for build & packaging. But this is something we can handle in a separate PR.